fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeActivity.kt

```diff
@@ -51,8 +51,12 @@ class HomeActivity : AppCompatActivity() {
     // When back to the home activity set connecting in view model to false
     viewModel.disableConnectingFlag()
 
+    viewModel.observeInternetConnection()
+
     handleTopAppBar()
 
+    observeInternetConnection()
+
```

Comment:
I'd avoid to have 2 function calls semantically so close, so you can call that  `viewModel.observeInternetConnection` in the `observeInternetConnection `method